docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,841 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit.algorithms.minimum_eigensolvers.vqe.VQEResult at 0x28d0a7ee0>"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results.relaxed_result"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The result of the rounding scheme is also worth considering. In this example, we used the `SemideterministricRounding`. It's important to note that with semi-deterministic rounding, a single sample is generated as the result, making it the optimal solution candidate.\n",
+    "\n",
+    "However, if we use the `MagicRounding` instead, multiple samples would be generated, each with a probability associated with it. These probabilities sum up to one, providing a distribution of potential optimal solutions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[SolutionSample(x=array([0, 1, 1, 0, 1, 0]), fval=5.0, probability=1.0, status=<OptimizationResultStatus.SUCCESS: 0>)]"
+      ]
+     },
+     "execution_count": 8,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "results.samples"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exact Problem Solution with the `NumpyMinimumEigensolver`\n",
+    "\n",
+    "To assess the performance of QRAO in approximating the optimal solution, we can utilize the `NumpyMinimumEigensolver`, an exact classical optimizer. We can obtain the exact optimal solution to the problem as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "objective function value: 9.0\n",
+      "variable values: x_0=0.0, x_1=1.0, x_2=0.0, x_3=1.0, x_4=1.0, x_5=0.0\n",
+      "status: SUCCESS\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit.algorithms.minimum_eigensolvers import NumPyMinimumEigensolver\n",
+    "\n",
+    "from qiskit_optimization.algorithms import MinimumEigenOptimizer\n",
+    "\n",
+    "exact_mes = NumPyMinimumEigensolver()\n",
+    "exact = MinimumEigenOptimizer(exact_mes)\n",
+    "exact_result = exact.solve(problem)\n",
+    "print(exact_result.prettyprint())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The approximation ratio (QRAO's approximate optimal function value divided by the exact optimal function value) tells us how closely QRAO approximated the optimal solution to the problem."
```

Comment:
"function value" -> "objective function value"

Reply:
"optimal" infers "exact". So, you don't need to say "exact optimal". "optimal" is enough. 
On the other hand, QRAO's solution is approximate. So, it enough to say "QRAO's objective function value".
My suggestion is as follows: "The approximation ratio (QRAO's objective function value divided by the optimal objective function value)"